templates/partners/silicon/index.html

```diff
@@ -22,350 +25,478 @@
+                output_mode="attrs"
+            )
+          }
+        },
+        {
+          "type": "description",
+          "item": {
+            "type": "html",
+            "content": '
+                <p>
+                    Accelerate design wins by developing joint, cost-effective solutions - Canonical\'s software portfolio, on the latest silicon, reduces total cost of ownership for end customers. Joint marketing increases silicon reach across ever growing Ubuntu and other open source communities.
+                </p>
+
+                <p>
+                    Take advantage of a clear established route for partners (ODM, OEM, etc) to enable, certify, optimize, promote, and sell silicon with Canonical software.
+                </p>
 
-  <section class="p-section">
-    <div class="row">
-      <hr class="p-rule" />
-      <h2>Expand opportunities through joint marketing and co-selling with Canonical</h2>
+                <p>
+                    Expand into new domains by leveraging and enhancing projects that Canonical has already successfully delivered for our enterprise and cloud customers.
+                    Read our stories and start the journey with Canonical now.
+                </p>
+            '
+          }
+        },
+        {
+          "type": "list",
+          "item": {
+            "list_items": [
+                {
+                  "list_item_type": "bullet",
+                  "content": '
+                    <a href="https://ubuntu.com/engage/partners-benefits-of-working-with-canonical-in-iot">
+                        Access the whitepaper: Scale, accelerate, and reduce time-to-market throughout IoT development
+                    </a>
+                  '
+                },
+                {
+                  "list_item_type": "bullet",
+                  "content": '
+                    <a href="https://ubuntu.com/engage/kubernetes-by-canonical-delivered-on-nvidia-dgx-systems">
+                      Download our solution brief and learn more about Kubernetes by Canonical delivered on NVIDIA DGX systems
+                    </a>
+                  '
+                }
+            ]
+          }
+        }
+      ]
+    ) }}
     </div>
-    <div class="row">
-      <div class="col-9 col-start-large-4">
-        <div class="p-strip is-shallow u-no-padding--top">
-          <hr class="p-rule" />
-        </div>
-        <div class="row">
-          <div class="col-3 u-hide--medium u-hide--small">
-            <div class="p-card--silicon-v-suru">
-              {{ image(url="https://assets.ubuntu.com/v1/5c68bbf0-ill-3.svg",
-                            alt="handshake illustration on dark suru backgroud",
-                            width="55",
-                            height="41",
-                            hi_def=True,
-                            loading="lazy") | safe
-              }}
+
+    <section class="p-section">
+        <div class="grid-row--50-50-on-large">
```

Comment:
It looks like in the future we might need to introduce a split on large variant of the basic pattern for cases like this.